src/_pytest/main.py

```diff
@@ -478,6 +479,9 @@ def __init__(self, config: Config) -> None:
         self.trace = config.trace.root.get("collection")
         self._initialpaths: FrozenSet[Path] = frozenset()
 
+        self._pm = config.pluginmanager
```

Comment:
I think we can remove this attribute variable in favor of the local variable in `gethookproxy` as before, as it is really being used there. 👍